test003.c: 函数 `test00' :
test003.c:10: エラー: `p' に対する代入演算子が不正です.
test003.c: 函数 `test01' :
test003.c:18: エラー: `p' に対する代入演算子が不正です.
